### PR TITLE
[BTRX-102][risk=no] Update ontology's swagger implementation

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses 
https://broadinstitute.atlassian.net/browse/BTRX-102

### CODE REVIEW

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java
Comment (vvicario) [inline src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: I think we can use StringUtils.isNotEmpty to verify if content is null or empty

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java
Comment (vvicario) [inline src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java:17]: what about if we move this path to config file, because if we update swagger version we will need to update this class

**Review by vvicario — APPROVED** [review]
Hi @rushtong code looks great I made a suggestion for swagger path.

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java
Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java:17]: I like that idea, but would like to hold off on that until some other PRs are merged. Will follow up with a story for it.

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java
Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: We can, thank you!

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java
Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java:17]: Also, would like to tackle this for consent too.

**Pull request merged**
